backup-recovery.adoc

```diff
@@ -1,26 +1,101 @@
+
+* `btc_mainnet/db/AccountAgeWitnessStore` (salt)
+* `btc_mainnet/keys/sig.key` (key used to sign p2p messages)
+* `btc_mainnet/keys/UserPayload` (account details)
+
+**Again, please make sure you don't have any open offers, trades, or disputes before changing `sig.key`.**
+
+IMPORTANT: Before manually making any changes to Bisq data files, please remember to back up the current state of your Bisq data directory and make sure the Bisq program is closed.
+
+== Tor State
+
+Bisq runs as a Tor hidden service to protect your privacy.
+
+As you do trades, you'll build _local reputation_ whereby Bisq will show you how many trades you've completed with other peers. The idea is that if one trade with someone went well, it's reasonable to expect your next trade with that person will go well too, creating a rudimentary sort of reputation while maintaining anonymity. This mechanism is tied to your onion address.
+
+If you want to migrate a particular onion address, you can replace the `btc_mainnet/tor/hiddenservice` folder in your data directory with the one from your backup.
+
+If you want to change your onion address, just delete the `btc_mainnet/tor/hiddenservice`. The next time you start Bisq, you'll have a different onion address.
+
+**Make sure you don't have any open offers, trades, or disputes before changing your onion address, as doing so will make you unreachable.**
+
+IMPORTANT: Before manually making any changes to Bisq data files, please remember to back up the current state of your Bisq data directory and make sure the Bisq program is closed.
+
+== Wallet Data
+
+Your wallet data is stored in `btc_mainnet/wallet`.
+
+Wallet issues (e.g., incorrect balance showing, corrupted/out-of-sync SPV file) can often be solved by resyncing the SPV chain and restarting the Bisq program. You can do this by clicking the `Delete SPV file and resync` button on the `Network info` tab of the `Settings` screen. If you can't access this screen, you can just delete the `bisq.spvchain` file in `btcmainnet/wallet` (just be advised that you will need to manually restart Bisq twice to complete the resync).
+
+If issues persist, you can try to restore a backup wallet file from `btc_mainnet/wallet/backup/backups_bisq_BTC_wallet`. This folder contains 20 of your most recent wallet backups (backups are made on a rolling basis). Replace the `bisq_BTC.wallet` wallet file in `btc_mainnet/wallet` with a backup wallet file and rename it to `bisq_BTC.wallet`. Start Bisq and see if your issue is fixed. If not, repeat with another backup.
+
+In extreme cases, if neither resyncing the SPV file nor restoring a wallet backup solves the problem, you might need to set up a new data directory. Make sure you don't have any open offers, trades, or disputes before doing this, as you won't be able to respond to these activities any longer.
```

Comment:
Hm... I think if the wallet is completely fucked up then restore from seed words is the emergency tool to get the funds restored. Switching to to a new data dir is more if there are stange bugs. There is a wallet db file which is important for the internal wallet management (db/AddressEntryList). If that is out of sync best is to start over with a new data dir.